src/pages/LoginPage/LoginPage.tsx

```diff
@@ -0,0 +1,306 @@
+import { Box, Typography, useTheme, Link, Button, IconButton, type Theme } from '@mui/material';
+import React, { useState } from 'react';
+
+import FacebookStatic from '../../assets/facebook.svg';
+import GithubStatic from '../../assets/github.svg';
+import GoogleStatic from '../../assets/google.svg';
+
+import {
+  getPageContainerStyles,
+  getFormStyles,
+  getLogoStyles,
+  getInputStyles,
+  getButtonStyles,
+  forgotPasswordLinkStyles,
+  signUpLinkStyles,
+  orYouCanTextStyles,
+  getSocialIconStyles,
+  legalTextStyles,
+  legalLinkStyles,
+  getErrorTextStyles,
+  GEOMETRY_PX,
+} from './LoginPage.styles';
+
+import type { LoginPageProps } from './LoginPage.types';
+import type { SxProps } from '@mui/material/styles';
+
+type LogoFallbackProps = {
+  orientation?: string;
+  width?: string | number;
+  height?: string | number;
+  alt?: string;
+};
+
+type FormInputFallbackProps = {
+  label?: string;
+  value?: string;
+  onChange?: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  placeholder?: string;
+  errorMessage?: string;
+  sx?: SxProps<Theme>;
+  type?: string;
+};
+
+type PrimaryButtonFallbackProps = {
+  children?: React.ReactNode;
+  onClick?: (e: React.FormEvent) => void;
+  disabled?: boolean;
+  loading?: boolean;
+  sx?: SxProps<Theme>;
+};
+
+const FacebookIcon: string | undefined = FacebookStatic ?? undefined;
+const GithubIcon: string | undefined = GithubStatic ?? undefined;
+const GoogleIcon: string | undefined = GoogleStatic ?? undefined;
+
+const Logo: React.ComponentType<LogoFallbackProps> = () => <div data-testid="logo-fallback" />;
```

Comment:
Logo fallback currently renders a bare <div data-testid="logo-fallback" />. This lacks semantic markup and accessible name. If the logo conveys brand identity it should be an <img> (with alt) or an inline <svg> with a title.

**Sugerencia:**
```typescript
Render a semantic element (e.g., <Box component="img" src={...} alt={alt}/>), or at minimum add role="img" and aria-label. Also forward width/height props from LogoFallbackProps into the rendered element.
```